templates/squid.conf.header.erb

```diff
@@ -10,6 +10,13 @@ memory_cache_shared           <%= (@memory_cache_shared == 'on' || @memory_cache
 <% end -%>
 maximum_object_size_in_memory <%= @maximum_object_size_in_memory %>
 
+<% unless @cache_replacement_policy.nil? -%>
```

Comment:
Since these are `Strings` and not `Booleans`, you can simplify this to just
```
<% if @cache_replacement_policy -%>
```

Reply:
everything done, something else to handle or ready to merge?

Reply:
Hi @SourceDoctor the comment @alexjfisher about the simpler `if` rather than `unless .. nil?` still stands.